Wczytanie_danych.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
Oczekiwałbym na początku jednej komórki ze wszystkimi importami z notatnika. pozwoli to monitorować zależności notatnika.